Blish HUD/GameServices/Modules/UI/Views/ManagePkgView.cs

```diff
@@ -0,0 +1,220 @@
+                    _statusImage.BasicTooltipText = "A newer version of this module is available.";
+                    _statusImage.Visible          = true;
+                    break;
+                case PkgVersionRelationship.CurrentVersion:
+                    _statusImage.Texture          = GameService.Content.GetTexture("common/157330");
+                    _statusImage.BasicTooltipText = "This module is installed and up to date.";
+                    _statusImage.Visible          = true;
+                    break;
+            }
+        }
+
+        private PkgVersionRelationship _versionRelationship = PkgVersionRelationship.NotInstalled;
+
+        public PkgVersionRelationship VersionRelationship {
+            get => _versionRelationship;
+            set {
+                if (_statusImage == null) throw new ViewNotBuiltException();
+
+                _versionRelationship = value;
+
+                SetRelationIcon(value);
+            }
+        }
+
+        private Label          _nameLabel;
+        private Label          _authLabel;
+        private Label          _descLabel;
+        private Image          _statusImage;
+        private Dropdown       _versionDropdown;
+        private StandardButton _actionButton;
+
+        public ManagePkgView() { /* NOOP */ }
+
+        public ManagePkgView(IGrouping<string, PkgManifest> model) {
+            this.WithPresenter(new ManagePkgPresenter(this, model));
+        }
+
+        protected override void Build(Panel buildPanel) {
+            _nameLabel = new Label() {
+                Text           = "W",
+                AutoSizeWidth  = true,
+                AutoSizeHeight = true,
+                Font           = GameService.Content.DefaultFont18,
+                Location       = new Point(5, 5),
+                Parent         = buildPanel
+            };
+
+            _authLabel = new Label() {
+                Visible           = false,
+                AutoSizeWidth     = true,
+                AutoSizeHeight    = false,
+                Height            = _nameLabel.Height,
+                VerticalAlignment = VerticalAlignment.Bottom,
+                Left              = _nameLabel.Right + 10,
+                Top               = _nameLabel.Top,
+                Parent            = buildPanel
+            };
+
+            _versionDropdown = new Dropdown() {
+                Width   = 128,
+                Right   = buildPanel.Width - 5,
+                Top     = 5,
+                Parent  = buildPanel
+            };
+
+            _statusImage = new Image(GameService.Content.GetTexture("common/157397")) {
+                Visible          = false,
+                Size             = new Point(16, 16),
+                Top              = _versionDropdown.Height / 2 - 8 + _versionDropdown.Top,
+                Right            = _versionDropdown.Left           - 8,
+                BasicTooltipText = "A newer version of this module is available.",
```

Comment:
localization support is missing here too

Reply:
256177e Thanks!